docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment, depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system, from which we can install the basic components of the AiiDA infrastructure in an isolated environment:
+
+.. code-block:: console
+
+    $ conda create -n aiida -c conda-forge aiida-core aiida-core.services
+    $ conda activate aiida
+    $ reentry scan
+
+.. note::
+
+    You can also install ``aiida-core`` from `PyPi <https://pypi.org/project/aiida-core>`_
```

Comment:
I think we should be more generic here and just say "Click here for alternative installation methods.". This is for users who were directed to this page (e.g. from a Stack Overflow answer or some other source), but actually don't want to use conda.